pkg/api/types.go

```diff
@@ -189,6 +189,9 @@ type VolumeSource struct {
 	// GCEPersistentDisk represents a GCE Disk resource that is attached to a
 	// kubelet's host machine and then exposed to the pod.
 	GCEPersistentDisk *GCEPersistentDiskVolumeSource `json:"gcePersistentDisk"`
+	// AWSPersistentDisk represents a AWS EBS disk that is attached to a
+	// kubelet's host machine and then exposed to the pod.
+	AWSPersistentDisk *AWSPersistentDiskVolumeSource `json:"awsPersistentDisk"`
```

Comment:
Perhaps.  EBS is just a marketing term though, right?  I'd rather think about the API in terms of what the user is trying to achieve - they want a persistent volume.  Maybe it should be AWSPersistentVolume.

I guess the question is when Amazon announces ElasticNvram, which is still a block device, would we create a second entry in the API?  (Or, if they had chosen to launch SSD under a different name).  I'd argue we shouldn't, and as such we shouldn't mirror Amazon's trademark-du-jour...